ops/model.py

```diff
@@ -822,8 +832,10 @@ def __getitem__(self, relation_name: str) -> List['Relation']:
         if not isinstance(relation_list, list):
             relation_list = self._data[relation_name] = []  # type: ignore
             for rid in self._backend.relation_ids(relation_name):
+                if rid == self._broken_relation_id:
+                    continue
                 relation = Relation(relation_name, rid, is_peer,
-                                    self._our_unit, self._backend, self._cache)
+                                    self._our_unit, self._backend, self._cache, True)
```

Comment:
Let's make the meaning clearer by using the keyword arg here (and below):

```suggestion
                                    self._our_unit, self._backend, self._cache, active=True)
```

Or perhaps we could make the arg default to `active=True` (seems reasonable) and leave it off here?

Reply:
Ha, [you said to not give it a default here](https://github.com/canonical/operator/pull/958/files#r1252495867) :smile:. I originally had it with a default and changed when I saw that comment - your point about it catching all the places it's called is valid, although I think that is probably just in the `ops` code and that's covered now.

I do feel that `active=True` is actually the default state and `active=False` is the unusual case, so a default of `True` makes sense to me. I guess current-Ben plus me outweighs previous-Ben :joy:.

Reply:
Ha, that's hilarious. Previous Ben had some good reasoning. But yeah, let's go with current-Ben + Tony. :-) Thanks.